dockertest/images.py

```diff
@@ -26,17 +26,15 @@
 # pylint: disable=W0403
 
 import re
+from autotest.client import utils
 from autotest.client.shared import error
 from config import Config
 from config import none_if_empty
 from config import get_as_list
-from autotest.client import utils
-from output import OutputGood
-from output import TextTable
+from output import OutputGood, TextTable
 from subtestbase import SubBase
-from xceptions import DockerTestError
+from xceptions import DockerTestError, DockerCommandError
 from xceptions import DockerFullNameFormatError
-from xceptions import DockerCommandError
```

Comment:
C0412: 33,0: : Imports from package autotest are not grouped
  C0412: 34,0: : Imports from package dockertest are not grouped